Slim/Schema.pm

```diff
@@ -1403,6 +1403,52 @@ sub _createComments {
 	}
 }
 
+sub _createWork {
+	my ($self, $work, $composerID, $create) = @_;
+
+	if ( $work ) {
+		# Using native DBI here to improve performance during scanning
+		my $dbh = Slim::Schema->dbh;
+
+		if (!$create) {
+#			my $sth_delete = $dbh->prepare_cached( qq{
+#				DELETE FROM works
+#				WHERE id = ?
+#			} );
+#
+#			$sth_delete->execute( $trackId );
+		}
+
+		my $titlesort = Slim::Utils::Text::ignoreCaseArticles( $work );
+		my $titlesearch = Slim::Utils::Text::ignoreCase($work, 1);
+
+		my $sth = $dbh->prepare_cached('SELECT id FROM works WHERE titlesearch = ? AND composer = ?');
```

Comment:
I guess that's where you'd have to first search for the musicbrainz ID (if defined).

Reply:
I have studied the Musicbrainz works stuff, and find it ambiguous to the point of not wanting to use it.

Reply:
You've been warned! I'll send them your way 😬